mne/viz/_3d.py

```diff
@@ -532,6 +532,7 @@ def plot_alignment(
     fig=None,
     interaction="terrain",
     sensor_colors=None,
+    sensor_scales=None,
```

Comment:
We've been trying to add these to functions as we add new params. Arguably it should go higher up but this is at least some progress
```suggestion
    *,
    sensor_scales=None,
```